specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
In the future, the RLA calculations will presumably be based on separate samples for each county, which will probably happen with a different sampling rates in each county.
For now, `ballots_to_audit_list_table` sounds like the list that is presented to the county before they actually audit them, thus my confusion.  Changing the name to something like `audited_ballot_list_table` might make it clearer. But I'm good with this for this purpose and phase.